js-wixcode-sdk/$w/VideoBox.service.json

```diff
@@ -0,0 +1,608 @@
+          [ { "lineno": 164,
+              "filename": "VideoBox.es6" } ],
+        "docs":
+          { "summary": "Sets or gets the video box volume.",
+            "description":
+              [ "Setting the `volume` property sets the video box's percentage of total",
+                " volume. Set the value to a number between `0` and `100`.",
+                "",
+                " Getting the `volume` property returns the video box's current",
+                " percentage of total volume." ],
+            "links": [],
+            "examples":
+              [ { "title": "Get a video box's volume",
+                  "body":
+                    [ "let volume = $w(\"#myVideoBox\").volume // 42" ],
+                  "extra":
+                    {  } },
+                { "title": "Set a video box's volume",
+                  "body":
+                    [ "$w(\"#myVideoBox\").volume = 42;" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "extra":
+          { "notes":
+              [ "The actual volume heard by a site visitor depends on the value of the\n `volume` property and the volume settings on the visitor's device." ] } } ],
+  "operations":
+    [ { "name": "mute",
+        "labels": [],
+        "nameParams": [],
+        "params": [],
+        "ret":
+          { "type":
+              { "name": "Promise",
+                "typeParams":
+                  [ "void" ] },
+            "doc": "Fulfilled - When the volume has been muted." },
+        "locations":
+          [ { "lineno": 346,
+              "filename": "VideoBox.es6" } ],
+        "docs":
+          { "summary": "Mutes video box volume.",
+            "links": [],
+            "examples":
+              [ { "title": "Mute volume",
+                  "body":
+                    [ "$w(\"#myVideoBox\").mute();" ],
+                  "extra":
+                    {  } },
+                { "title": "Mute volume and log a message when done",
+                  "body":
+                    [ "$w(\"#myVideoBox\").mute()",
+                      "  .then( () => {",
+                      "    console.log(\"Done with mute\");",
+                      "  } );" ],
+                  "extra":
+                    {  } } ],
+            "extra":
+              {  } },
+        "extra":
+          {  } },
+      { "name": "onEnded",
+        "labels": [],
+        "nameParams": [],
+        "params":
+          [ { "name": "handler",
+              "type": "$w.EventHandler",
+              "doc": "The name of the function or\n the function expression to run when the video has ended." } ],
+        "ret":
+          { "type": "$w.VideoBox",
+            "doc": "The video box that triggered the event." },
+        "locations":
+          [ { "lineno": 506,
+              "filename": "VideoBox.es6" } ],
+        "docs":
+          { "summary": "Adds an event handler that runs when playback has ended.",
```

Comment:
I think playback should have the word 'the' before it (all over the API). I realize we have it without 'the' for video player, but could be it's incorrect.

Reply:
I will open a doc bug